docs/topics/data-analysis/lets-plot.md

```diff
@@ -0,0 +1,200 @@
+[//]: # (title: Data visualization with Lets-Plot for Kotlin)
+
+[Lets-Plot for Kotlin (LPK)](https://lets-plot.org/kotlin/get-started.html) is a multiplatform plotting library that serves as a port of [R's ggplot2 library](https://ggplot2.tidyverse.org/) to
+Kotlin. LPK brings the feature-rich ggplot2 API to the Kotlin ecosystem,
+making it suited for scientists and statisticians who require sophisticated data visualization capabilities.
+
+LPK targets various platforms, including [Kotlin notebooks](data-analysis-overview.md#notebooks), [Kotlin/JS](js-overview.md), Kotlin [JVM Swing](https://docs.oracle.com/javase/8/docs/technotes/guides/swing/), [JavaFX](https://openjfx.io/), and [Kotlin Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/).
+Additionally, LPK has seamless integration with [IntelliJ](https://www.jetbrains.com/idea/), [DataGrip](https://www.jetbrains.com/datagrip/), [Datalore](https://datalore.jetbrains.com), [DataSpell](https://www.jetbrains.com/dataspell/), and [PyCharm](https://www.jetbrains.com/pycharm/).
+
+![Lets-Plot](lets-plot-overview.png){width=700}
+
+This tutorial demonstrates how to create different plot types with
+the LPK and [Kotlin DataFrame](https://kotlin.github.io/dataframe/gettingstarted.html) libraries, using Kotlin Notebook in IntelliJ IDEA.
+
+## Before you start
+
+1. Download and install the latest version of [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/download/?section=mac).
+2. Install the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook) in IntelliJ IDEA.
+
+   > Alternatively, access the Kotlin Notebook plugin from **Settings** | **Plugins** | **Marketplace** within IntelliJ IDEA.
+   >
+   {type="tip"}
+
+3. Create a new notebook by selecting **File** | **New** | **Kotlin Notebook**.
+4. In your notebook, import the LPK and Kotlin DataFrame libraries by running the following command:
+
+    ```kotlin
+    %use lets-plot
+    %use dataframe
+    ```
+
+## Prepare the data
+
+Create a DataFrame that stores simulated numbers of the monthly average temperature in three cities: Berlin, Madrid, and Caracas.
```

Comment:
I personally find it strange to have a data frame instance capitalized like it's a product name, but I see that the same approach is used on the Data analysis visualization page. So this questions is broader that this PR and can be resolved separately

Reply:
Yes, this was something I had talked with @daniCsorbaJB about, and the dev team requested him to manage it like this. We can discuss further.